test-lib.sh

```diff
@@ -1134,7 +1134,7 @@ ct_run_tests_from_testset() {
   for test_case in $TEST_SET; do
     TESTCASE_RESULT=0
     # shellcheck disable=SC2076
-    if [[ " ${UNSTABLE_TESTS[*]} " =~ " ${test_case} " ]]; then
+    if [[ " ${UNSTABLE_TESTS[*]} " =~ " ${app_name} " ]]; then
```

Comment:
I guess we need to check also if UNSTABLE_TESTS is defined or not.

Reply:
Hi, this is my fault, I pushed the fixing commit here, please check if it is correct, or if not just discard it.